c/test/check_bitfield_macros.c

```diff
@@ -56,6 +56,8 @@ END_TEST
 
 START_TEST(test_ins_status_bitfields)
 {
+// TODO(?) : This test case currently failing
```

Comment:
Was this failing prior to this PR?

Reply:
Yes I didn't dig in to exactly why but it fails when enabled on master as well

Reply:
I am trying to get up to speed on this since all these bitfield macros were my doing; when I check master, this test passes and is enabled.  
Code: https://github.com/swift-nav/libsbp/blob/master/c/test/check_bitfield_macros.c#L57
CI: https://travis-ci.com/github/swift-nav/libsbp/jobs/478975927#L1620

Am I missing something?

Reply:
`test_imu_bitfields` and `test_ins_status_bitfields` weren't added in to the test suite on master so they weren't being run. See the change on lines 77-78